sdk/src/androidTest/java/com/bugsnag/android/ErrorStoreTest.java

```diff
@@ -123,6 +123,19 @@ public void isStartupCrash() throws Exception {
         assertFalse(errorStore.isStartupCrash(10000));
     }
 
+    @Test
+    public void testFindOldFiles() throws Throwable {
+        new File(errorStore.oldDirectory, "foo.json").createNewFile();
+        File dir = errorStore.storageDir;
+        new File(dir, "foo.json").createNewFile();
+
+        List<File> files = errorStore.findStoredFiles();
+        for (File file : files) {
+            assertTrue(file.getAbsolutePath().endsWith("foo.json"));
+        }
+        assertEquals(2, files.size());
```

Comment:
If there are multiple clients, will all attempt to find and send files from the "old" directory?

Reply:
Yes. I've added a mazerunner scenario which covers this.